libbeat/monitoring/report/elasticsearch/client.go

```diff
@@ -20,6 +20,9 @@ package elasticsearch
 import (
 	"encoding/json"
 	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
did you mean to import `errors` or `pkg/errors`?

Reply:
`pkg/errors` as I'm using it for `errors.Wrap`.